test-lib.sh

```diff
@@ -538,8 +577,14 @@ EOF
     else
         echo "CMD /usr/libexec/s2i/run" >>"$df_name"
     fi
+
+    if $ca_file; then
```

Comment:
I guess this can now be removed and mount options set unconditionally?

Reply:
I will try and test it.